### PR TITLE
release v1.0.0-beta.3

### PR DESCRIPTION
**Pull request by weekface**
### What problem does this PR solve? 

- Release TiDB Operator v1.0.0-beta.3
- adjust stability parameters

### Check List 

Tests 

 - No code

Code changes

 - Has documents change

Side effects

Related changes

### Does this PR introduce a user-facing change?:
 
 ```release-note
NONE
 ```

### CODE REVIEW

File: tests/actions.go [hunk 1]
```diff
@@ -1921,7 +1921,7 @@ func (oa *operatorActions) CheckScheduledBackup(info *TidbClusterConfig) error {
 
 		succededJobCount := 0
 		for _, j := range backupJobs {
-			if j.Status.Failed > 0 {
+			if j.Status.Failed > 3 {
```
Comment (aylei) [inline tests/actions.go]: Are these changes intended?

Comment (weekface) [reply tests/actions.go]: Yes, the pod may fail one or two times.

File: tests/actions.go [hunk 2]
```diff
@@ -96,7 +96,7 @@ func NewOperatorActions(cli versioned.Interface,
 const (
 	DefaultPollTimeout          time.Duration = 10 * time.Minute
 	DefaultPollInterval         time.Duration = 1 * time.Minute
-	BackupAndRestorePollTimeOut time.Duration = 30 * time.Minute
+	BackupAndRestorePollTimeOut time.Duration = 60 * time.Minute
```
Comment (weekface) [inline tests/actions.go]: We need a longer restore period.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,13 +1,93 @@
+# TiDB Operator v1.0.0-beta.3 Release Notes
+
+## v1.0.0-beta.3 What’s New
+
+### Action Required
+- ACTION REQUIRED: `nodeSelectorRequired` was remove from values.yaml.
```
Comment (tennix) [inline CHANGELOG.md]:
```suggestion
- ACTION REQUIRED: `nodeSelectorRequired` was removed from values.yaml.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,13 +1,93 @@
...
+- ACTION REQUIRED:  Comma-separated values support in `nodeSelector` has been dropped, please use new-added `affinity` field which has a more expressive syntax.
+
+### A lot of stability cases added
+- ConfigMap rollout
+- One PD replicas
+- Stop TiDB Operator itself
+- TiDB stabile scheduling
```
Comment (tennix) [inline CHANGELOG.md]:
```suggestion
- TiDB stable scheduling
```
